packages/SwingSet/src/controller.js

```diff
@@ -345,17 +347,25 @@ export async function buildVatController(
     return new Worker(supercode);
   }
 
+  // launch a worker in a subprocess (which runs Node.js)
+  function startSubprocessWorkerNode() {
+    const supercode = require.resolve(
+      './kernel/vatManager/subprocessSupervisor.js',
+    );
+    return startSubprocessWorker(process.execPath, ['-r', 'esm', supercode]);
```

Comment:
The use of `process.execPath` here seems like it could introduce some interesting (and I'm not sure it it's the good kind of interesting or the bad kind of interesting) possibilities if the executable is something that wraps node or otherwise is not actually node itself.  In particular, I can imagine a use case in which the launching executable is something greater than node per se but where the workers really should be run with the actual node executable.  I'm not sure this is a problem we need to solve now or indeed if it's even a problem at all, but you might give some thought to putting in a way to parameterize what gets run here instead of assuming "run whatever kind of thing I am".

Reply:
yeah, I figured this was the simplest way to run under the same version of Node as the parent process, but needing to add the `-r esm` is a hassle. Once we get rid of `-r esm` and move to `type: module` everywhere, maybe this could change.